plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -115,6 +122,11 @@ const renderTabs = (
 					key={ tabKey }
 				>
 					{ tabs[ tabKey ]?.title }
+					{ tabs[ tabKey ]?.showUpdateCount && updateCount > 0 && (
```

Comment:
The `showUpdateCount`, while being a tab property, defines visibility of a value `updateCount` that is outside a tab context. How about moving `updateCount` into the tab context as well?

Reply:
Fixed with: https://github.com/woocommerce/woocommerce/pull/46088/commits/f09891b9b7f1beca9088881dae7d969b2bf50bc9